scripts/config.py

```diff
@@ -133,6 +133,15 @@ def full_adapter(name, value, active):
         return active
     return include_in_full(name)
 
+def realfull_adapter(_name, _value, _active):
```

Comment:
You need to add something in custom_parser_options in class TFPSACryptoConfigTool as well I think.